config/translation-checker.php

```diff
@@ -0,0 +1,20 @@
+<?php
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Laravel Translation Checker
+    |--------------------------------------------------------------------------
+    |
+    | This package allows you to check your Laravel translations for missing
+    | keys, unused keys and unused translations.
+    | You can ignore some keys by adding them to the 'ignore' array.
```

Comment:
I know this is the description i used in my PR but currently the package let's you find missing keys but don't have any functionality related to "unused keys/translations". When I opened the PR I just pasted an example description without paying too much attention. 
Maybe you can remove that part to better reflect the current functionality of the package.